client/extensions/woocommerce/app/store-stats/constants.js

```diff
@@ -115,3 +115,10 @@ export const UNITS = {
 		title: translate( 'Years' )
 	}
 };
+
+export const chartTabs = [
+	{ label: translate( 'Gross Sales' ), attr: 'gross_sales', type: 'currency' },
+	{ label: translate( 'Net Sales' ), attr: 'net_sales', type: 'currency' },
+	{ label: translate( 'Orders' ), attr: 'orders', type: 'number' },
```

Comment:
Hi! I've found a possible matching string that has already been translated **100** times:
`translate( 'Order' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/WCS-stats-analytics)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).